keps/sig-release/2572-release-cadence/README.md

```diff
@@ -0,0 +1,440 @@
+- [Proposal](#proposal)
+  - [User Stories](#user-stories)
+    - [End User](#end-user)
+    - [Distributors and downstream projects](#distributors-and-downstream-projects)
+    - [Contributors](#contributors)
+    - [SIG Release members](#sig-release-members)
+  - [Risks and Mitigations](#risks-and-mitigations)
+    - [Concentrating risk](#concentrating-risk)
+    - [Attention to tests](#attention-to-tests)
+    - [Attention to dependencies](#attention-to-dependencies)
+    - [Feature graduation](#feature-graduation)
+    - [Unprepared Kubernetes Users](#unprepared-kubernetes-users)
+    - [Increased enhancements lifecycle](#increased-enhancements-lifecycle)
+- [Design Details](#design-details)
+  - [Schedule Policy](#schedule-policy)
+  - [Feedback survey](#feedback-survey)
+- [Implementation History](#implementation-history)
+  - [GitHub Discussion](#github-discussion)
+  - [Leads meeting feedback session](#leads-meeting-feedback-session)
+- [Drawbacks](#drawbacks)
+- [Alternatives](#alternatives)
+<!-- /toc -->
+
+## Release Signoff Checklist
+
+- [ ] (R) Enhancement issue in release milestone, which links to KEP dir in [kubernetes/enhancements] (not the initial KEP PR)
+- [ ] (R) KEP approvers have approved the KEP status as `implementable`
+- [ ] (R) Design details are appropriately documented
+- [ ] (R) Test plan is in place, giving consideration to SIG Architecture and SIG Testing input (including test refactors)
+- [ ] (R) Graduation criteria is in place
+- [ ] (R) Production readiness review completed
+- [ ] (R) Production readiness review approved
+- [ ] "Implementation History" section is up-to-date for milestone
+- [ ] User-facing documentation has been created in [kubernetes/website], for publication to [kubernetes.io]
+- [ ] Supporting documentation—e.g., additional design documents, links to mailing list discussions/SIG meetings, relevant PRs/issues, release notes
+
+[kubernetes.io]: https://kubernetes.io/
+[kubernetes/enhancements]: https://git.k8s.io/enhancements
+[kubernetes/kubernetes]: https://git.k8s.io/kubernetes
+[kubernetes/website]: https://git.k8s.io/website
+
+## Summary
+
+With this KEP, SIG Release proposes to change the current Kubernetes release
+cadence from 4 down to 3 releases per year. This cadence started in ad-hoc manner
+in 2020, and will be formalized with the KEP. This will be reflected in the 
+release calendar for the Kubernetes 1.22 and 1.23 releases, which will each be 
+*15* weeks in duration.
+
+## Motivation
+
+Discussions around changing the release cadence for Kubernetes, which currently
+releases 4 times per year, are ongoing in the community.
+
+The extended release schedule for 1.19 resulted in only three minor Kubernetes
+releases for 2020. As a result, SIG Release received several questions across a
+variety of platforms and communication channels about whether the project
+intends to only have three minor releases/year.
```

Comment:
`releases/year` -> `releases/year as a lot of folks need to plan ahead (both developers and end users)` (expand a bit if appropriate?

Reply:
added!